util/include/Timers.hpp

```diff
@@ -0,0 +1,127 @@
+// Concord
+//
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include <functional>
+#include <stdint.h>
+#include <chrono>
+#include <boost/heap/binomial_heap.hpp>
+
+namespace concordUtil {
+
+class TimerHeap;
+
+class Timer {
+ public:
+  class greater {
```

Comment:
Lower-case class name? Does it have to be a class with an overloaded operator? Why don't you define a static function in the timer class?
I feel like I'm missing something ...

Reply:
Removed, as it's no longer needed.